js/banner/banner.encryption.js

```diff
@@ -7,11 +7,21 @@
 ( function ( Banner ) {
 	'use strict';
 
-	var EP;
+	var EP,
+		browserSupportsCryptoAPI = function () {
+			if ( typeof window !== 'undefined' && window.crypto && window.crypto.getRandomValues ) {
+				return true;
+			} else if ( typeof window !== 'undefined' && typeof window.msCrypto === 'object'
+				&& typeof window.msCrypto.getRandomValues === 'function' ) {
+				return true;
+			}
+			return false;
+		};
 
 	function Encryption() {
 		var self = this;
 		this.initialized = false;
+		this.useLegacyEncryption = !browserSupportsCryptoAPI();
```

Comment:
`useLegacyEncryption` is only used in `initCryptLib`, isn't it? If only used there, how about making it a local variable of that method?